packages/fetch/src/fetch.ts

```diff
@@ -29,26 +23,33 @@ export function fetch(info: RequestInfo, requestInit: RequestInit = {}): Fetch {
   }
 
   return {
+    name: `fetch \`${info}\``,
+    labels: {
+      expand: false
+    },
```

Comment:
What about representing some of the `requestInit` inside the labels? It seems to me that parameters like `method` are extremely handy to have.

Reply:
Good idea!